sys/net/gnrc/network_layer/ipv6/gnrc_ipv6.c

```diff
@@ -530,26 +530,29 @@ static inline kernel_pid_t _next_hop_l2addr(uint8_t *l2addr, uint8_t *l2addr_len
                                             kernel_pid_t iface, ipv6_addr_t *dst,
                                             gnrc_pktsnip_t *pkt)
 {
+    kernel_pid_t found_iface;
```

Comment:
See fixup (and I did this because otherwise cppcheck would have been complaining)